recipes-kernel/linux/linux-mainline/0008-sun8i-h2-plus-orangepi-zero.dts-enable-spidev.patch

```diff
@@ -21,31 +25,29 @@ index 494f49a3e9f0..cc4377e73e5d 100644
  
  	flash@0 {
  		#address-cells = <1>;
-@@ -199,11 +199,24 @@
+@@ -199,11 +199,23 @@
  		compatible = "mxicy,mx25l1606e", "winbond,w25q128";
  		reg = <0>;
  		spi-max-frequency = <40000000>;
-+		status = "disabled";
-+	};
-+
-+	spidev@0 {
-+		compatible = "spidev";
++    status = "disabled";
```

Comment:
@PLangowski please use correct indents. It does not look the same as in old patch

Reply:
Done 05a1da4c10b5589d555dd379d398d589facf0475